README.md

```diff
@@ -1,34 +1,27 @@
-# ewpe-smart-mqtt
-MQTT bridge for EWPE Smart powered devices which can be controled via WiFi using [EWPE Smart app](https://play.google.com/store/apps/details?id=com.gree.ewpesmart)
+# proklima-to-mqtt
```

Comment:
Looks like you forgot to revert this back

Reply:
ah sorry totally messed up with the branche.

plan to publish a docker image for "Bauhaus Proklima" devices. its not clear that they use the same API.

i'will fix it and cleanup the pull request